### PR TITLE
More accurate already concluded error

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: jacquard/experiments/tests/test_smoke.py [hunk 1]
```diff
@@ -66,6 +66,42 @@ def test_conclude_updates_defaults():
     assert config.storage['defaults'] == BRANCH_SETTINGS
 
 
+def test_conclude_before_launch_not_allowed():
+    config = Mock()
+    config.storage = DummyStore('', data=DUMMY_DATA_PRE_LAUNCH)
+
+    stderr = io.StringIO()
+    with contextlib.redirect_stderr(stderr), pytest.raises(SystemExit):
+            main(('conclude', 'foo', 'bar'), config=config)
```
Comment (prophile) [inline jacquard/experiments/tests/test_smoke.py]: This looks like it's indented too far?

File: jacquard/experiments/commands.py [hunk 2]
```diff
@@ -115,11 +115,17 @@ def handle(self, config, options):
             concluded_experiments = store.get('concluded-experiments', [])
 
             if options.experiment not in current_experiments:
-                raise CommandError(
-                    "Experiment '{experiment_id}' not launched!".format(
+                if experiment.concluded is None:
+                    message = "Experiment '{experiment_id}' not launched!".format(
```
Comment (prophile) [inline jacquard/experiments/commands.py]: This line is too long.

File: jacquard/experiments/commands.py [hunk 3]
```diff
@@ -115,11 +115,17 @@ def handle(self, config, options):
...
                         experiment_id=options.experiment,
-                    ),
-                )
+                    )
+                else:
+                    message = "Experiment '{experiment_id}' already concluded (at {concluded})!".format(
```
Comment (prophile) [inline jacquard/experiments/commands.py]: This line is also too long.

**Comment by prophile** [comment]
Test failure is down to a dependency change; if you would be so kind as to rebase onto master that'll resolve itself.

LGTM other than review points, thanks for doing this! 👍

File: jacquard/experiments/tests/test_smoke.py (hunk 1 above)
Comment (PeterJCLaw) [reply jacquard/experiments/tests/test_smoke.py]: 5d66336

File: jacquard/experiments/commands.py (hunk 2 above)
Comment (PeterJCLaw) [reply jacquard/experiments/commands.py]: b726ea6

File: jacquard/experiments/commands.py (hunk 3 above)
Comment (PeterJCLaw) [reply jacquard/experiments/commands.py]: b726ea6

**Comment by prophile** [comment]
Thanks!

**Pull request merged**